scripts/ssm_install.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Install AWS SSM agent for different OS versions
+os_type=$(uname -m)
+os_release=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
+
+if [ "$${os_type}" == "x86_64" ]; then
```

Comment:
This looks great and I can see you have make sure to cover the architecture, I wonder though, if you had a variable with the architecture could you not use the if statements?

Reply:
I have not clearly comprehended your question here